test/specs/ExpectedTextSpec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+import { BrowserUtils } from '../../src/commons/BrowserUtils';
+import { describeCommon } from '../TestHelper';
+
+const DIV_DOUBLE_CLICK: string = "//*[@id='div-double-click']";
+
+/**
+ * wdio-allure-ts expectedText tests
+ */
+describeCommon('ExpectedText of BrowserUtils Tests', () => {
+  it('Validate expectedText Change ', () => {
+    BrowserUtils.scrollIntoView(DIV_DOUBLE_CLICK);
+    BrowserUtils.expectText(DIV_DOUBLE_CLICK, 'Double click');
+    BrowserUtils.doubleClick(DIV_DOUBLE_CLICK);
+    BrowserUtils.expectText(DIV_DOUBLE_CLICK, 'Double Click Event');
```

Comment:
doubleClick in html does not have sleep timeout. So it changes immediately. We need to make sure the polling of "new" text will work.

Reply:
Done